StorageUtility/Linux/Device.h

```diff
@@ -19,10 +19,6 @@ limitations under the License.
 #define __Device_h__

```

Comment:
The different about device obj between Win and Linux :
- Device obj must inherit from DeviceInterface
- The first, this is support of API of os in order to create Device obj.
- The second, Data structure of Device object are diff. On Linux Device is descript by spec files, and Handle is Interger... On Windown, handle is void pointer
- Device object is used create Drive. It is dependent API of os